lib/packagekit-glib2/pk-task.c

```diff
@@ -58,13 +56,15 @@ struct _PkTaskPrivate
 enum {
 	PROP_0,
 	PROP_SIMULATE,
-	PROP_ONLY_PREPARE,
+	PROP_ONLY_DOWNLOAD,
```

Comment:
Why was this renamed?

Reply:
The property is actually named "only-download" and the setter/getter are named set/get_only_download so It makes sense to have it consistent here too

Reply:
This has no implication on API/ABI

Reply:
I didn't think it did, since you didn't change the order in the enumeration.